lib/src/input_qty.dart

```diff
@@ -285,7 +285,7 @@ class _InputQtyState extends State<InputQty> {
           keyboardType: TextInputType.number,
           inputFormatters: [
             // LengthLimitingTextInputFormatter(10),
-            FilteringTextInputFormatter.allow(RegExp(r'^\d*\.?\-?\d*')),
+            FilteringTextInputFormatter.allow(RegExp(r'^\d*\.?\d*')),
```

Comment:
f we remove the -?\ from the pattern, we will not be able to type negative number anymore.